gradle/scripts/README.md

```diff
@@ -213,7 +213,7 @@ boms:
 applied so you can conveniently check if your dependencies are out of date:
 
 ```
-$ ./gradlew dependencyUpdates -Drevision=release
+$ ./gradlew dependencyManagement:dependencyUpdates
```

Comment:
How about adding an alias task to the root project?

Reply:
Went ahead and just removed the path so the task is run when there's no colon. Should we add an alias too or is this ok?

Reply:
I think it's OK.